docs/kintoblocks.md

```diff
@@ -43,6 +43,28 @@ The steps to tag a commit are:
 
 The main reason to tag a KintoBlock is to add it to an application, think of it as a release, so when you have your KintoBlock finished and want to update it in an application you can tag it and select the tag in the application. This is a way to control the released versions of your KintoBlock.
 
-## Custom Parameters (Coming Soon)
-## Environment Parameters (Coming Soon)
+## Environment Variables
+
+Environment Variables are used to add values required for your KintoBlock to work out in the world. These are often configurations like  `CLIENT_IDS` and `SECRETS` that you do not want to expose in your code. Inside your KintoBlock you acess these via `env` variables. 
```

Comment:
To work out in the world => need to be more technical.

- Environment variables are used to control static information that you want to control in the application.
- Environment variables are static between multiple environments.  You cannot change an environment variables between applications and their environments.
- In the near future configuration management will be used to configure your application as a per-environment use case.

I think we need to discuss this so its more clear - cuz client ids / secrets should be config params, not environment variables.

Reply:
Example would be like private keys.